src/pages/Teletubbies/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from "react";
+import { Container, Typography, Grid, Paper } from "@mui/material";
+import teletubbiesData from "../../Data/teletubbies.json";
+
+const Teletubbies = () => {
+  useEffect(() => {}, []);
+
+  return (
+    <>
+      <Container>
+        <Grid>
+          <Typography
+            variant="h2"
+            sx={{
+              border: "2px solid",
+              borderColor: "primary.main",
+              color: "Blue",
+            }}
+            mt={4}
+            mb={3}
+          >
+            Teletubbies
+          </Typography>
+        </Grid>
+        {teletubbiesData.map((teletubby, index) => (
```

Comment:
`teletubby` doesn't have an id?

Reply:
Because there isn't in teletubbies.json.